lib/t/cli.rb

```diff
@@ -688,7 +688,7 @@ def retweets_of_me
     end
     map %w(retweetsofme) => :retweets_of_me
 
-    desc 'ruler', 'Prints a 140-character ruler'
+    desc 'ruler', 'Prints a 280-character ruler'
     method_option 'indent', aliases: '-i', type: :numeric, default: 0, desc: 'The number of spaces to print before the ruler.'
     def ruler
       markings = '----|'.chars.cycle.take(140).join
```

Comment:
You didn't actually update the ruler functionality to print 280 chars — you need to update the `140` to `280` here as well.